VideoLocker/res/layout/view_progress_download_btn.xml

```diff
@@ -15,16 +14,9 @@
         android:layout_width="25dp"
         android:layout_height="25dp"
```

Comment:
I changed these to use the dimen resources, but not the course outline row layouts, as those need to be standardized all over the layout, and are out of scope here.